src/xmidtsend_rbus.c

```diff
@@ -64,7 +64,7 @@ void addToXmidtUpstreamQ(wrp_msg_t * msg, rbusMethodAsyncHandle_t asyncHandle)
 	{
 		char * errorMsg = strdup("Max Queue Size Exceeded");
 		ParodusError("Queue Size Exceeded\n");
-		createOutParamsandSendAck(msg, asyncHandle, errorMsg , QUEUE_SIZE_EXCEEDED);
+		createOutParamsandSendAck(msg, asyncHandle, errorMsg , QUEUE_SIZE_EXCEEDED, RBUS_ERROR_INVALID_INPUT);
```

Comment:
@shilpa24balaji Max queue size is invalid input from t2 ? or it should be an error >100 with "RBUS_ERROR_INVALID_RESPONSE_FROM_DESTINATION" as it happened due to client disconnected for too long. 

https://github.com/xmidt-org/parodus/blob/master/src/xmidtsend_rbus.h#L54

Reply:
Ya lets map max queue size as RBUS_ERROR_INVALID_RESPONSE_FROM_DESTINATION, as it is the response from parodus not being able to process due to wan down etc.

Reply:
Resolved in latest change